sky/tools/gn

```diff
@@ -126,13 +120,10 @@ def parse_args(args):
   args = args[1:]
   parser = argparse.ArgumentParser(description='A script run` gn gen`.')
 
-  parser.add_argument('--debug', default=True, action='store_true')
-  parser.add_argument('--release', default=False, dest='debug', action='store_false')
+  parser.add_argument('--debug', default=False, action='store_true')
```

Comment:
It's probably worth an email to the team about changing the default from Debug to Release and about renaming the out directories.